starters/cms/src/FieldMapping.tsx

```diff
@@ -132,9 +132,17 @@ export function FieldMapping({ collection, dataSource, initialSlugFieldId }: Fie
         })
     }
 
+    const isAllowedToManage = useIsAllowedTo("ManagedCollection.setFields", ...syncMethods)
+
     const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
         event.preventDefault()
 
+        if (!isAllowedToManage) {
+            console.error("Insufficient permissions for management")
+            framer.notify("Insufficient permissions for management", { variant: "error" })
+            return
+        }
```

Comment:
I would make the form disabled, otherwise you might think you can make changes while you can't. That's too late because it's a frustrating experience

Reply:
I looked into it when making this PR, there isn't really a way to "disable" a form sans disabling all the submission triggers (like the submit button, which is done). This is a safeguard against some "submit on enter press" behavior of some browser. I didn't turn it into just a `return` because you'd still want to know what happened if you rely on that behavior of your browser.

Reply:
So I would just return

Reply:
I feel like it's a bad move to rely on just disabling things. Disabled inputs can still be focusable. Can also forget to disable something. Turning this into a `return` would be trading robustness for brevity, is that worth it? Also, see my comment about not wanting to disable the rest of the UI: https://github.com/framer/plugins/pull/211#discussion_r2037029630